### ISSUE TITLE
Encode the AndroidManifest.xml

### ISSUE THREAD

**Issue by alterechtschreibung** [ytsutano/axmldec#8]
How can I encode the AndroidManifest.xml again? This would be very useful.

**Comment by daedroza** [comment]
Indeed, I would like to see this feature too as long as we have the private keys. @alterechtschreibung A easier workaround method would be to use apksigner and your private key to resign it.

**Comment by alterechtschreibung** [comment]
Even  just the possibility to convert it to binary format again would be very useful.

**Comment by wearehosted** [comment]
do we know if this has been achieved to encode the manifest back once the changes have been made?

**Comment by evgenybozhko2** [comment]
Still active, what is the way to encode it back after changes ?
